ui/component/publishForm/view.jsx

```diff
@@ -293,27 +294,61 @@ function PublishForm(props: Props) {
   }, [modal]);
 
   // move this to lbryinc OR to a file under ui, and/or provide a standardized livestreaming config.
-  function fetchLivestreams(channelId, signature, timestamp) {
```

Comment:
### Crash when choosing replay
I streamed live using the new API ...stopped... and tried to publish the replay. It crash.
FWIW, I do have some legacy replays, and 1 "inprogress"

<img width="263" alt="image" src="https://user-images.githubusercontent.com/64950861/155063263-378cb6d8-35b9-4dcd-b575-aed07e86bad0.png">

<img width="972" alt="image" src="https://user-images.githubusercontent.com/64950861/155063343-f287b40c-e1b3-4ba3-99d7-0dead4ec9fb6.png">